src/xdocs/cmdline.xml.vm

```diff
@@ -73,6 +73,14 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-o file</code> - specify the file to output
           to.
         </li>
+        <li>
+          <code>-s</code> - print xpath suppressions at the file's line and column position. The option
```

Comment:
`-s` is missing the argument after it, and you must describe the format of the argument in detail.

Reply:
Done